pkg/server/server.go

```diff
@@ -271,7 +271,7 @@ func (s *ProxyServer) removeBackend(agentID string, conn agent.AgentService_Conn
 }
 
 func (s *ProxyServer) addFrontend(agentID string, connID int64, p *ProxyClientConnection) {
-	klog.V(2).InfoS("Register frontend for agent", "frontend", p, "agentID", agentID, "connectionID", connID)
+	klog.V(5).InfoS("Register frontend for agent", "frontend", p, "agentID", agentID, "connectionID", connID)
```

Comment:
Note to reviewers: this is largely redundant with https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/6faac346f5c7144b3802ad266a3878859442bdab/pkg/server/server.go#L805-L810

Reply:
if so let's just delete this one.